model-optimizer/extensions/front/tf/non_max_suppression_normalize.py

```diff
@@ -64,18 +64,34 @@ def replace_sub_graph(graph: Graph, match: dict, **kwargs):
                                                                 {'name': nms.soft_get('name') + '/Unsqueeze_1'})
         nms.in_port(1).get_connection().insert_node(unsqueeze_box_scores)
 
-        # prepare output
-        crop_box_indices = Crop(graph, {'name': nms.soft_get('name') + '/Crop', 'axis': int64_array([1]),
+        nms_name = nms.soft_get('name', nms.id)
+
+        # prepare output #0
+        crop_box_indices_name = nms_name + '/Crop_boxes_'
+        crop_box_indices = Crop(graph, {'name': crop_box_indices_name, 'axis': int64_array([1]),
                                         'offset': int64_array([2]), 'dim': int64_array([1])}).create_node()
         nms.out_port(0).get_connection().insert_node(crop_box_indices)
         squeeze_output_boxes = create_op_node_with_second_input(graph, Squeeze, int64_array([1]),
-                                                                {'name': crop_box_indices.soft_get('name') + '/Squeeze'}
-                                                                )
+                                                                {'name': crop_box_indices_name + '/Squeeze'})
         crop_box_indices.out_port(0).get_connection().insert_node(squeeze_output_boxes)
 
-        if 5 in nms.in_ports() and not nms.in_port(5).disconnected():
-            soft_nms_sigma = nms.in_port(5).get_source().data.get_value()
-            if soft_nms_sigma is not None and soft_nms_sigma != 0.0:
-                log.error('The input to layer "{}" with value for the soft_nms_sigma is equal to "{}" but only value 0'
-                          'is supported. The inference results will be incorrect.'.format(nms.soft_get('name'),
-                                                                                          soft_nms_sigma))
+        num_of_outputs = len([port for port in nms.out_ports().values() if not port.disconnected()])
+
+        if num_of_outputs == 1:
+            return
+
+        # prepare output #1
+        crop_score_indices_name = nms_name + '/Crop_scores_'
+        crop_score_indices = Crop(graph, {'name': crop_score_indices_name, 'axis': int64_array([1]),
+                                          'offset': int64_array([2]), 'dim': int64_array([1])}).create_node()
+        nms.out_port(1).get_connection().insert_node(crop_score_indices)
+        squeeze_output_scores = create_op_node_with_second_input(graph, Squeeze, int64_array([1]),
+                                                                 {'name': crop_score_indices_name + '/Squeeze'})
+        crop_score_indices.out_port(0).get_connection().insert_node(squeeze_output_scores)
+
+        # make inputs 2 to 6 to have shape [1] instead of [0] (convert 0D to 1D)
+        for port_id in range(2, 6):
```

Comment:
This is change is already done in the `model-optimizer/extensions/front/onnx/non_max_suppression_normalize.py`

Reply:
The transformation from the file `model-optimizer/extensions/front/onnx/non_max_suppression_normalize.py` performs this change only for ONNX models, but the transformation from the file `model-optimizer/extensions/front/tf/non_max_suppression_normalize.py` performs changes only for TF models.

Reply:
Move `model-optimizer/extensions/front/onnx/non_max_suppression_normalize.py` to `model-optimizer/extensions/front/non_max_suppression_normalize.py` so it will be applied for any framework models and remove the duplicate code in this file.

Reply:
Done.